### DISCUSSION TITLE
How to test a trained model?

### DISCUSSION THREAD

**Discussion by dee6600** [utiasDSL/gym-pybullet-drones#116] [Q&A] [answered]
Thank you for putting this project together. I am new to RL projects. 

I have trained a model with default parameters on the singleagent.py script for hover. And I will appreciate if someone tells me how to test the trained model in sim. I mean see in action! I am not able to figure out where the inference script or test script is. I am playing with fly.py and script.py but I am struggling. Please help. Thank you.

**Comment by dee6600** [comment, marked as answer]
got it!

**Reply by AR-AK** [reply]
Could you please tell how to achieve that?

**Reply by piratax007** [reply]
Maybe this https://github.com/piratax007/gym-pybullet-drones/tree/main/gym_pybullet_drones/experiments could be helpful. There, I have separated scripts for learning and simulation.
